src/pagopa_proxy.tf

```diff
@@ -97,7 +97,7 @@ module "pagopa_proxy_app_service" {
     REDIS_DB_URL      = module.pagopa_proxy_redis[0].hostname
     REDIS_DB_PORT     = module.pagopa_proxy_redis[0].ssl_port
     REDIS_DB_PASSWORD = module.pagopa_proxy_redis[0].primary_access_key
-    REDIS_USE_CLUSTER = true
+    REDIS_USE_CLUSTER = var.env_short == "p"
```

Comment:
why not create a new variable? related to the env?
for example, we could also decide in UAT to set this parameter to true (maybe 😁)